apps/meteor/client/views/marketplace/UnlimitedAppsUpsellModal.tsx

```diff
@@ -1,17 +1,30 @@
 import { useSetting, useTranslation } from '@rocket.chat/ui-contexts';
-import { FlowRouter } from 'meteor/kadira:flow-router';
-import React from 'react';
+import React, { useCallback } from 'react';
 
 import UpsellModal from '../../components/UpsellModal';
 
-const UnlimitedAppsUpsellModal = () => {
+export type UnlimitedAppsUpsellModalProps = {
+	onClose: () => void;
+};
+
+const UnlimitedAppsUpsellModal = ({ onClose }: UnlimitedAppsUpsellModalProps) => {
 	const t = useTranslation();
 	const cloudWorkspaceHadTrial = useSetting('Cloud_Workspace_Had_Trial') as boolean;
 	const urls = {
 		goFullyFeaturedRegistered: 'admin/upgrade/go-fully-featured-registered',
-		talkToSales: 'go.rocket.chat/i/contact-sales',
+		talkToSales: 'https://www.rocket.chat/sales-contact',
```

Comment:
We need to keep the `go.rocket.chat` URL here, it helps us visualize access
```suggestion
		talkToSales: 'https://go.rocket.chat/i/contact-sales',
```